Map_Filter_Fold/Delegates.Tests/Delegates.cs

```diff
@@ -0,0 +1,51 @@
+namespace Delegates.Tests;
+using MapFilterFold;
+public class Tests

```

Comment:
```suggestion
namespace Delegates.Tests;

using MapFilterFold;

public class Tests

```
код с отступами в нужных местах легче читается :)